mne/defaults.py

```diff
@@ -11,22 +11,26 @@
                ref_meg='steelblue', misc='k', stim='k', resp='k', chpi='k',
                exci='k', ias='k', syst='k', seeg='saddlebrown', dipole='k',
                gof='k', bio='k', ecog='k', hbo='#AA3377', hbr='b',
-               fnirs_cw_amplitude='k', fnirs_od='k', csd='k'),
+               fnirs_cw_amplitude='k', fnirs_fd_ac_amplitude='k',
+               fnirs_fd_phase='k', fnirs_od='k', csd='k'),
     units=dict(mag='fT', grad='fT/cm', eeg='µV', eog='µV', ecg='µV', emg='µV',
                misc='AU', seeg='mV', dipole='nAm', gof='GOF', bio='µV',
                ecog='µV', hbo='µM', hbr='µM', ref_meg='fT',
-               fnirs_cw_amplitude='V', fnirs_od='V', csd='mV/m²'),
+               fnirs_cw_amplitude='V', fnirs_fd_ac_amplitude='V',
+               fnirs_fd_phase='V', fnirs_od='V', csd='mV/m²'),
```

Comment:
Should phase be in degree or radians?

Reply:
degrees, I wasn't sure where to put this

Reply:
I think I commented where below (if I got this review thing working)

Reply:
We should convert it to radians, too, as all data in MNE should be in SI units. We can convert from radians to degrees for display purposes much like we convert V to uV for EEG, etc.